pom.xml

```diff
@@ -203,8 +203,9 @@
                     <version>${maven.release.plugin.version}</version>
                     <configuration>
                         <autoVersionSubmodules>true</autoVersionSubmodules>
-                        <preparationGoals>clean verify install</preparationGoals>
+                        <preparationGoals>clean install</preparationGoals>
                         <pushChanges>false</pushChanges>
+                        <arguments>-Prelease ${arguments}</arguments>
```

Comment:
Looks like we do have a couple of libraries (e.g., `org.codice.alliance.libs.klv`) and api packages (e.g., `org.codice.alliance.imaging.chip.service.api`) that are being exported. Not sure if those are for public consumption though.

@shaundmorris, should we hold off on the JavaDoc generation until we figure out what we want to include, or should we just get it included now and figure it out later?